### ISSUE TITLE
What val_loss value you get?

### ISSUE THREAD

**Issue by KUASWoodyLIN** [qqwweee/keras-yolo3#132]
hello everyone

I will like to know what loss value you get?
What is the **normal val_loss range**?
And how to evaluation loss size?

here is setting:
Step 1 : fine-turn on last layer
- epochs=50
- batch_size = 32
- optimizer = Adam, lr=1e-3

Step 2 : fine-turn on all layer
- epochs=50
- batch_size = 8
- optimizer = Adam, lr=1e-4

and here my loss value traing on training on BDD dataset:
**the last value is: 62.07**
and it take 6 day to training!!!
![val_loss](https://user-images.githubusercontent.com/9446515/41893441-50fe7430-794e-11e8-97a7-173c1b7163f7.png)

**Comment by sherlockchou86** [comment]
my training is not finished, but the last val_loss is about 30.0 after 5 epochs(i stop the training process because i got a new machine with higher performance GPU :) )

parameters : 50 epochs, 8 batch_size, optimizer and lr are default. 

i will use my new machine to train my data set, update the comment soon.

**Comment by KUASWoodyLIN** [comment]
@sherlockchou86 
Look like your training val_loss can be very low!!!
I think the bdd dataset ground truth is too difficult to found, that cause my training loss too big.

**Comment by cylinbao** [comment]
I trained on VOC dataset and got around 16 on val_loss after 120 epoch.
The trained model can achieve 77% mAP if I lower confidence threshold to 0.005.

**Comment by kwea123** [comment]
I trained a model with 1 class, got loss of 24. The performance is **significantly** lower than what I get using tensorflow object detection...

A same image using yolo/tensorflow object detection on the SAME training set (OID)
The detection threshold is 0.0 (showing everything it detected)

yolo:
![yolo](https://user-images.githubusercontent.com/11364490/42720526-82b58464-8763-11e8-80a4-d03b57768b8a.png)

tensorflow:
![tensorflow](https://user-images.githubusercontent.com/11364490/42720552-d0f7199e-8763-11e8-94f7-a6d547a8bdd2.png)

**Comment by kt-smitha** [comment]
@kwea123 , i want to train on single class to detect person more accurately, for which i changed the class.txt to person and then am i supposed to change the .cfg before converting to get .h5

The change such as the filter and class number. ?

**Comment by eain3314** [comment]
50左右 效果很差
